vllm/worker/habana_model_runner.py

```diff
@@ -1388,7 +1397,8 @@ def warmup_graphs(self, strategy, buckets, is_prompt, kv_caches,
 
     @torch.inference_mode()
     def warmup_model(self, kv_caches: List[torch.Tensor]) -> None:
-        if os.environ.get('VLLM_SKIP_WARMUP', 'false').lower() == 'true':
+        # fix(huijong): just skip warmup for now
+        if self.scheduler_config.chunked_prefill_enabled or os.environ.get('VLLM_SKIP_WARMUP', 'false').lower() == 'true':
```

Comment:
As [1d prefill can be performed without chunking](https://github.com/SqueezeBits/vllm-fork/pull/7) and warmups have no effect with 1d-prefill-only case either, it's better to change `if self.scheduler_config.chunked_prefill_enabled or` to `if self.scheduler_config.enable_1d_prefill or`.